pkg/schema/schema.go

```diff
@@ -238,6 +238,9 @@ func (s *Node) UnmarshalYAML(node *yaml.Node) error {
 	if node != nil && node.Kind == yaml.DocumentNode {
 		return s.UnmarshalYAML(node.Content[0])
 	}
+	if node.Kind == yaml.AliasNode {
```

Comment:
```suggestion
	if node != nil && node.Kind == yaml.AliasNode {
```

to match the `if` above so you don't crash on `nil`